nx-X11/lib/X11/Xlibint.h

```diff
@@ -880,10 +953,6 @@ extern void _XEatData(
     Display*		/* dpy */,
     unsigned long	/* n */
 );
-extern void _XEatDataWords(
```

Comment:
@uli42: the _XEatDataWords prototype should not get removed here. You later on add it in 13be6c792902dbb5a728ad2ec431f6ed80aa678d. Please drop the remove/add commits from this and the referenced commit (commit msg: "Xlibint.h: add missed part").